src/modules/SerialModule.cpp

```diff
@@ -252,7 +252,8 @@ void SerialModule::sendTelemetry(meshtastic_Telemetry m)
         pb_encode_to_bytes(p->decoded.payload.bytes, sizeof(p->decoded.payload.bytes), &meshtastic_Telemetry_msg, &m);
     p->to = NODENUM_BROADCAST;
     p->decoded.want_response = false;
-    p->priority = meshtastic_MeshPacket_Priority_RELIABLE;
+    p->want_ack = true;
+    p->priority = meshtastic_MeshPacket_Priority_MAX;
```

Comment:
Please don't set it higher than `ACK`, because those need to get priority always to avoid excessive retransmissions and flooding. You could use `meshtastic_MeshPacket_Priority_HIGH`.

Reply:
also what about framing this change with device.role.sensor? because that would be the appropriate role for this kind of packets.

Reply:
I changed to high from max.  The sensor role could work but the ws85 sensor node can't go to sleep or it'll miss the serial wind data that comes in every 2 seconds.  Sensor role could muck that up i think.

Reply:
Sensor role should only sleep if you have power.is_power_saving enabled

Reply:
> also what about framing this change with device.role.sensor? because that would be the appropriate role for this kind of packets.
> 
> E.G. use 'reliable' and no want_ack from all other roles...

are you proposing that I check for sensor_role here and only set want_ack and high priority if sensor_role is set ?